backend/tests/unit/test_main.py

```
@@ -0,0 +1,805 @@
+"""
+Comprehensive tests for the main FastAPI application.
+
+Tests lifespan management, endpoints, WebSocket connections, and background processing.
+"""
+
+import asyncio
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)